sklego/preprocessing/pandastransformers.py

```diff
@@ -19,6 +20,15 @@ class ColumnDropper(BaseEstimator, TransformerMixin):
     feature_names_ : list[str]
         The names of the features to keep during transform.
 
+    Notes
+    -----
+    Supported dataframes are:
```

Comment:
Consider adding a link to narwhals docs where the list of supported dataframe libraries is mentioned.
Currently I was able to spot it in [extending section](https://marcogorelli.github.io/narwhals/extending/) only.